### DISCUSSION TITLE
easy install on ubuntu

### DISCUSSION THREAD

**Discussion by eoli3n** [raysan5/raylib#2673] [Q&A]
I'm sysadmin, I need to automate deployment process for 800 hosts for a University.
Is there a way to package raylib for Ubuntu, as deb package with a ppa, or as flatpak ?

**Comment by AndyMender** [comment]
I think there are 2 options
- using `cpack` through cmake and distributing that
- building a native deb package for Ubuntu

This might be a good starting point for a native package - [ArchLinux PKGBUILD](https://github.com/archlinux/svntogit-community/blob/packages/raylib/trunk/PKGBUILD)

If you have your own provisioning setup (Ansible, Chef, etc.) you could build a deb locally and distribute that.

**Comment by eoli3n** [comment]
For now, I will build on a NFS export.

**Comment by faveoled** [comment]
Check out Ubuntu 22.04 PPA install:
https://github.com/raysan5/raylib/discussions/3295

**Comment by texus** [comment]
I've created a _libraylib5-dev_ PPA package for raylib 5.0 at [ppa:texus/raylib](https://launchpad.net/~texus/+archive/ubuntu/raylib) (supporting multiple Ubuntu versions)
```
sudo add-apt-repository ppa:texus/raylib
sudo apt install libraylib5-dev
```
Edit: I've also added a libraylib6-dev package in the PPA.

**Comment by JupiterRider** [comment]
There is also a request for the official Debian repository:
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1011133

**Comment by Wajinn** [comment]
It's wild after all these years and drama, still no .deb files in the official release or flatpak. At least Debian is slowly bringing something along in testing.
